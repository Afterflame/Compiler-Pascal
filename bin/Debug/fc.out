Сравнение файлов input.out и INPUT.ANS
FC: различия не найдены

