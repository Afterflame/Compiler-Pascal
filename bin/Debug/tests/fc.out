Сравнение файлов 7.out и 7.ANS
***** 7.out
Closing bracket expected at 1 32
***** 7.ANS
 +
 \-- *
 |   \-- +
 |   |   \-- a23
 |   |   \-- 14
 |   \-- 16
 \-- +
     \-- /
     |   \-- ff
     |   \-- 12
     \-- /
         \-- 14
         \-- *
             \-- 2
             \-- -
                 \-- 1
                 \-- 1
*****

